fhe-toolkit-macos/SampleCode/CapitalDetailViewController.mm

```diff
@@ -49,7 +49,7 @@ @implementation CapitalDetailViewController
 // Plaintext prime modulus
 unsigned long p = 131;
 // Cyclotomic polynomial - defines phi(m)
-//unsigned long m = 33;  // this will give 10 slots - runs faster
+//unsigned long m = 33;  // this will give 20 slots - runs faster
```

Comment:
```suggestion
```

@XuesongHu your comment  regarding 20 slots and not 10 is correct.
That line was a leftover from when we changed the algebra to work with up to 48 slots to accommodate for up to 48 chars country/capital names. `m=33` will not work for this demo, so this line should be deleted.

Also, we changed slightly the organization of the directories which will cause merge conflicts. Can you rebase your fork , please?

Thanks